testing/test_likelihoods.py

```diff
@@ -181,14 +181,45 @@ def testSymmetric(self):
             pred = tf.Session().run(l.predict_density(F, F, Y), feed_dict={x: l.get_free_state(), F: F_data})
```

Comment:
note how above that line 176 there is the `l.invlink.epsilon = epsilon`. This is fine as epsilon are the same before and after. However, if epsilon does change then `_eps_K1` will not change to suit. I don't know if you want me to make _eps_K1 a property therefore, or add a setter to epsilon?